packages/app/src/App.vue

```diff
@@ -5,23 +5,19 @@
     />
   </router-view>
 
-  <template v-if="route.name && route.name !== 'SpecRunner'">
+  <template v-if="!isRunMode">
     <!--
-      checking for existence of `route.name` here to avoid a flash
-      of these components if the page is refreshed on the SpecRunner route
```

Comment:
Is the "flash" behavior no longer an issue? I'm not seeing any weirdness doing a refresh in the runner, but not clear whether I'm just not replicating what this was guarding against or if it's really not a problem

Reply:
I discussed this with @marktnoonan . The comment was added after the check `route.name &&` was added to the beginning of the `v-if`.  The real reason for the check was to not run it when in run mode.  This check was added because when in the runner, the main app header is removed which uses these two components, thus they were not needed.  For the new Debug option in the sidebar, we need the `CloudViewerAndProject` loaded, so switching to use this `!isRunMode` check instead

Reply:
Yeah the comment was not well worded for the future, it took me a second to remember the bad side effect of the "flash" rendering of those components - which was, their setup functions ran, even on the runner page, which was not necessary in open mode, and actively needed to be avoided in run mode. Now it's necessary in open mode, and still avoided in run mode.